listlibs/listlibs.go

```diff
@@ -7,15 +7,16 @@ import (
 )
 
 // ListLoadedLibraries lists all the libraries (their absolute paths) loaded by a process.
-func ListLoadedLibraries(p process.Process) (libraries []string, harderror error, softerrors []error) {
+func ListLoadedLibraries(p process.Process) (libraries []string, softerrors []error, harderror error) {
 	return listLoadedLibraries(p)
 }
 
-// GetMaGetMatchingLoadedLibraries lists the libraries loaded by process p whose path matches r.
-func GetMatchingLoadedLibraries(p process.Process, r *regexp.Regexp) (libraries []string, harderror error,
-	softerrors []error) {
+// GetMatchingLoadedLibraries lists the libraries loaded by process p whose path matches r.
+func GetMatchingLoadedLibraries(p process.Process, r *regexp.Regexp) (libraries []string, softerrors []error,
+	harderror error,
+) {
```

Comment:
I did on every file but didn't realize that I should do after modification too. But worked upon it now.